### ISSUE TITLE
blob.download_to_filename() broken with 0.27.0

### ISSUE THREAD

**Issue by tcroiset** [GoogleCloudPlatform/google-cloud-python#3841]
I open the bug here since it's linked to the new release 0.27.0 (urrlib2 -> requests), though it crashes in dependency google-resumable-media-python

```
  File "/home/tristan/p/api/api/utilities/storage.py", line 21, in get_gcs_blob
    blob.download_to_filename(local_path)
  File "/home/tristan/env/api/local/lib/python2.7/site-packages/google/cloud/storage/blob.py", line 482, in download_to_filename
    self.download_to_file(file_obj, client=client)
  File "/home/tristan/env/api/local/lib/python2.7/site-packages/google/cloud/storage/blob.py", line 464, in download_to_file
    self._do_download(transport, file_obj, download_url, headers)
  File "/home/tristan/env/api/local/lib/python2.7/site-packages/google/cloud/storage/blob.py", line 418, in _do_download
    download.consume(transport)
  File "/home/tristan/env/api/local/lib/python2.7/site-packages/google/resumable_media/requests/download.py", line 101, in consume
    self._write_to_stream(result)
  File "/home/tristan/env/api/local/lib/python2.7/site-packages/google/resumable_media/requests/download.py", line 62, in _write_to_stream
    with response:
AttributeError: __exit__
```

**Comment by dhermes** [comment]
@tcroiset Thanks for reporting. I'm sorry to say this is the third or fourth such report (100% my fault this error is occurring).

The simple fix: make sure `requests >= 0.18.0`. We have releases `google-cloud-storage==1.3.2` (via #3814) to ensure the minimum version.

Sorry for the inconvenience.

**Comment by jasonmellone** [comment]
Still getting this as of date posting. Shouldn't [https://anaconda.org/conda-forge/google-cloud-storage](https://anaconda.org/conda-forge/google-cloud-storage) solve for this?
